pom.xml

```diff
@@ -2606,6 +2606,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>3.1.3</version>
+        <configuration>
+          <classFilesDirectory>${basedir}/target/scala-2.11/classes</classFilesDirectory>
+          <testClassFilesDirectory>${basedir}/target/scala-2.11/test-classes</testClassFilesDirectory>
+          <effort>Max</effort>
```

Comment:
@kiszk, btw do you roughly know how much time this PR increases in the build?

Reply:
I do not see large difference w and w/o `Max`

```
user	17m3.724s
user	17m18.628s

```

Reply:
Eh, how much does it increase the Maven build time by this whole PR then roughly?